examples/nlp/language_modeling/tuning/conf/megatron_gpt_peft_eval_config.yaml

```diff
@@ -105,16 +105,17 @@ model:
       drop_last: False
       context_key: ${data.train_ds.context_key}
       label_key: ${data.train_ds.label_key}
-      add_eos: ${data.train_ds.add_eos}
+      add_eos: False
       add_sep: ${data.train_ds.add_sep}
```

Comment:
Since there is no ${data.train_ds} in this config, how do the other two values get populated?

Reply:
The eval script loads PEFT weights (and the peft config from training time) This config is embedded into the training config so when OmegaConf tries to resolve the Config it will contain the `data.train_ds`

This is done to reduce change of user error (setting one thing at train time and another at test time).

Reply:
@hsiehjackson why do we want to hard-code false? seems like we can leave it as is.

Reply:
I revert back to the origin. @arendu why we need add_eos to be True for validation and test? I think we don't need this for generation and only need if we want to calculate loss precisely as training.